src/style.css

```diff
@@ -1,3 +1,73 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+
+  --blue: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  width: 100%;
+  height: 100%;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav {
+  text-transform: uppercase;
```

Comment:
may be will be better add this style for list element but not for all nav, maybe in the future you will have additional elements in nav and should be change this style